src/events/EventsSection.tsx

```diff
@@ -88,8 +167,8 @@ export const EventsSection = () => {
   const DetailCell = (event: EventRepresentation) => (
     <>
       <DescriptionList isHorizontal className="keycloak_eventsection_details">
-        {Object.keys(event.details!).map((k) => (
-          <DescriptionListGroup key={`detail-${event.time}-${event.type}`}>
+        {Object.keys(event.details!).map((k, index) => (
+          <DescriptionListGroup key={`detail-${index}`}>
```

Comment:
Index should be used as a key as a [last resort only](https://reactjs.org/docs/lists-and-keys.html#keys). In this case I believe you can use `k` as the key since it should be unique.
```suggestion
          <DescriptionListGroup key={k}>
```